src/components/MenuItem.js

```diff
@@ -30,6 +30,9 @@ const propTypes = {
 
     // Overrides the icon for shouldShowRightIcon
     iconRight: PropTypes.elementType,
+
+    /** If true, the icon will appear larger and have a circular grey background with a white fill */
+    emphasizeIcon: PropTypes.bool,
```

Comment:
NAB, this seems kind of specific modifier to add as a generic option for the `MenuItem` component. Unless we know we will be using this design again for other things in the future. If not, would maybe start with just allowing `createMenuIconEmphasized` style to be passed in as a generic prop.